static.go

```diff
@@ -17,13 +17,14 @@ import (
 
 // Static contains the parsed content for a single GTFS static message.
 type Static struct {
-	Agencies  []Agency
-	Routes    []Route
-	Stops     []Stop
-	Transfers []Transfer
-	Services  []Service
-	Trips     []ScheduledTrip
-	Shapes    []Shape
+	Agencies    []Agency
+	Routes      []Route
+	Stops       []Stop
+	Transfers   []Transfer
+	Services    []Service
+	Trips       []ScheduledTrip
+	Shapes      []Shape
+	Frequencies []Frequency
```

Comment:
I understand the package is not really consistent on this point. For example, by the same argument `[]ScheduledTrip` should be nested inside `Routes` instead of being a top level field .... any thoughts? Not a big deal either way tbh

Reply:
Awesome